addon/mixins/axis-titles.js

```diff
@@ -0,0 +1,174 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+  /**
+   * Toggle axis titles on/off
+   * @type {Boolean}
+   */
+  hasAxisTitles: false,
+
+  /**
+   * Title for the x axis
+   * @type {String}
+   */
+  xValueDisplayName: null,
+
+  /**
+   * Title for the y axis
+   * @type {String}
+   */
+  yValueDisplayName: null,
+
+  /**
+   * Computed title for the x axis, if the `hasAxisTitles` boolean is false
+   * `xAxisTitleDisplayValue` should be an empty string
+   * @type {String}
+   */
+  xAxisTitleDisplayValue: Ember.computed('hasAxisTitles', 'xValueDisplayName', function(){
+    return this.get('hasAxisTitles') ? this.get('xValueDisplayName') : '';
+  }),
+
+  /**
+   * Computed title for the x axis, if the `hasAxisTitles` boolean is false
+   * `yAxisTitleDisplayValue` should be an empty string
+   * @type {String}
+   */
+  yAxisTitleDisplayValue: Ember.computed('hasAxisTitles', 'yValueDisplayName', function(){
+    return this.get('hasAxisTitles') ? this.get('yValueDisplayName') : '';
+  }),
+
+  /**
+   * Computed left margin, if the `hasAxisTitles` is false this should be 0
+   * @type {Number}
+   */
+  horizontalMarginLeft: Ember.computed('hasAxisTitles', function(){
+    return this.get('hasAxisTitles') ? 20 : 0;
```

Comment:
Is it possible to specify `hasAxisTitles`, but *not* specify `horizontalMarginLeft`? Maybe someone doesn't want to specify `horizontalMarginLeft`, but this will make it `0`, instead of `null`